masterlist.yaml

```diff
@@ -10771,12 +10771,11 @@ plugins:
 
   - name: 'Spell Research - Even More Patches.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/61177/' ]
-    req: [ 'SpellResearch.esp' ]
     msg:
       - <<: *requiresX
         type: error
         subs: [ '[Spell Research v2.2+](https://www.nexusmods.com/skyrimspecialedition/mods/20983/)' ]
-        condition: 'file("SpellResearch.esp") and (checksum("SpellResearch.esp", 755423D7) or checksum("SpellResearch.esp", 63661836) or checksum("SpellResearch.esp", 860EFC3D))'
```

Comment:
In some cases you will have to use both a message and req, so that LOOT sorts the patch after the main file.
It's probably not important for this patch, but some people do complain when patches aren't after the main files.

We could probably change that message to a requires version x or greater.

Reply:
Good points, they completely slipped my mind. SRPCR already has Spell Research as a master, but the condition can be adjusted to avoid an excess of errors. An after should be enough for SRPC so that we can still communicate its specific requirement in a single message.